common/scripts/mesh-11s.sh

```diff
@@ -91,6 +91,7 @@ echo "sudo mesh $1 $2 $3 $4 $5 $6 $7 $8 $9 ${10} ${11}"
         help
       fi
 
+iw dev "$wifidev" set mesh_param mesh_fwding 1
```

Comment:
Should be:
iw dev "$wifidev" set mesh_param mesh_fwding 0

Reply:
fixed with [55e2be6](https://github.com/tiiuae/mesh_com/pull/132/commits/55e2be66d7497aa10eca90288344f5ef7da7db9e)